blocks/subscriptions-block/components/Cart/index.jsx

```diff
@@ -0,0 +1,108 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from '@arc-fusion/prop-types';
+import { useFusionContext } from 'fusion:context';
+import getProperties from 'fusion:properties';
+import getTranslatedPhrases from 'fusion:intl';
+import { PrimaryFont } from '@wpmedia/shared-styles';
+import useSales from '../useSales';
+import currency from '../../utils/currency';
+
+import Item from './item';
+
+import './styles.scss';
+
+const getPaymentInfo = ({
+  origin,
+  endpoint,
+  priceCode,
+}) => fetch(`${origin}${endpoint}${priceCode}/1/${new Date().getTime()}`, {})
+  .then((res) => res.json());
+
+const Cart = ({ offerURL }) => {
+  const { arcSite } = useFusionContext();
+  const phrases = getTranslatedPhrases(getProperties(arcSite).locale || 'en');
+
+  const [isFetching, setIsFetching] = useState(true);
+  const [cartItems, setCartItems] = useState();
+
+  const { api: { retail: { origin } } } = getProperties(arcSite);
```

Comment:
If you hit a site that is not configured ( eg: `the-globe` ) `retail` is missing and it errors out.

Reply:
Are we safe to make the assumption that the setting has to exist?

Reply:
I added the missing config objects to Fusion-News-Theme master branch

Reply:
Well, it has to exist for this to even work... so it's a hard requirement that anyone using this will need to configure either in one of the prop locations.